definitions/redux-actions_v0.9.x/flow_>=v0.23.x/redux-actions_v0.9.x.js

```diff
@@ -0,0 +1,5 @@
+declare module 'redux-actions' {
+  declare function createAction(type: string, payloadCreator?: Function, metaCreator?: Function): Function;
```

Comment:
Does it make sense to type `payloadCreator` as `(payload: mixed) => any`? (and if so, same for `metaCreator`?)